src/main/ngapp/accio/src/app/img-gallery/img-gallery.component.html

```diff
@@ -23,10 +23,74 @@
   <button type="submit" mat-button  id="upload-button">Upload</button>
 </form>
 
-<div id="gallery" *ngIf="displayImages">
+<br>
```

Comment:
Note: mat-forms being difficult with 'display: relative' here. UI team set the css up for the html, so the <br> is just a placeholder for when UI team wants to correct the display placement of the search bar against the upload bar